### ISSUE TITLE
Fix configure.ac for Bela

### ISSUE THREAD

**Issue by giuliomoro** [giuliomoro/pure-data#4]
Try to leave portaudio and ALSA in.

**Comment by giuliomoro** [comment]
Also make sure extras get compiled in

**Comment by giuliomoro** [comment]
Closed b26fef73852c8b3a1dfdac75cba696280b74e566.

You shall configure with 
```
CFLAGS="-I/usr/xenomai/include -I/root/Bela/include" LDFLAGS="-L/root/Bela/lib -L/usr/xenomai/lib" ./configure --enable-bela
```
